BUILD.md

```diff
@@ -236,6 +237,18 @@ Follow these steps to run the `Demo.py` testcase:
 A GUI is also available to run this demo.
 See [examples/apps/heart_disease_eval](examples/apps/heart_disease_eval)
 
+## <a name="staticanalysis"></a>Static Analysis
+To run lint checks on codebase, execute the following commands -
+```
+cd $TCF_HOME
+docker-compose -f docker-compose-lint.yaml up
+```
+If you want to run lint on selective modules, you need to pass the modules via `LINT_MODULES`. For example -
```

Comment:
Add an example to check all modules.

Reply:
That's implicit @danintel . Thanks to your feedback, I have mentioned it clearly now. :)